src/components/Story.js

```
@@ -34,18 +36,21 @@ export default class Story extends React.Component {
       }
     }
   }
+
   toggleMore = show => {
     this.setState({ showMore: show })
   }
+
   imageLoaded = () => {
     try {
       if (this.pauseId) clearTimeout(this.pauseId)
-      this.setState({loaded: true})
+      this.setState({ loaded: true })
       this.props.action('play', true)
     } catch (e) {
       console.log(e)
```

show console.log is it a good action to do? Maybe throw an error instead?